### PR TITLE
Add matrix strategy to ruby tests

### PR DESCRIPTION
**Pull request by julianrubisch**
# CI Enhancement

## Description

Version 3 seems to be not available yet, so converting this to a draft PR.

## Checklist

- [X] My code follows the style guidelines of this project
- [X] Checks (StandardRB & Prettier-Standard) are passing
- [X] This is not a documentation update

Please note that the best way to suggest changes or updates to the [documentation](https://docs.stimulusreflex.com) is to [join Discord](https://discord.gg/XveN625) and leave a note in the #docs channel. Any documentation updates posted as PRs cannot be accepted at this time. :heart:

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -41,12 +41,15 @@ jobs:
   ruby_test:
     name: Ruby Test Action
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: [2.6.x, 2.7.x, 3.0.x]
```
Comment (andrewmcodes) [inline .github/workflows/tests.yml:46]: You'll have to pin v3 for now.

**Review by andrewmcodes — CHANGES_REQUESTED** [review]
Looks right at a glance other than my comment. 

Fwiw this is what I added for Bridgetown 

```

strategy:
      matrix:
        ruby_version: [2.5.8, 2.6.6, 2.7.2]
    continue-on-error: ${{ endsWith(matrix.ruby, 'head') || matrix.ruby == 'debug' }}

```

File: .github/workflows/tests.yml (hunk 1 above)
Comment (julianrubisch) [reply .github/workflows/tests.yml]: Hmm didn’t work 😒

**Comment by leastbad** [comment]
@andrewmcodes does this look good to you?

**Review by andrewmcodes — APPROVED** [review]
@leastbad it does. 

Great work on this @julianrubisch. This would be a great thing to share with others!

**Pull request merged**